data/pigui/modules/system-view-ui.lua

```diff
@@ -19,11 +19,10 @@ local colors = ui.theme.colors
 local icons = ui.theme.icons
 
 local systemView
+if Game then systemView = Game.systemView end -- for hot-reload
```

Comment:
If this works, you can just write this as `local systemView = Game and Game.systemView`

Reply:
Indeed, thanks!